components/Spinner/RouteSpinner.js

```diff
@@ -0,0 +1,28 @@
+export default function RouteSpinner() {
+  return (
+    <div className="flex fixed z-50 top-0 h-[100%] w-[100%] bg-black bg-opacity-50">
+      <div
+        role="status"
+        className="absolute -translate-x-1/2 -translate-y-1/2 top-2/4 left-1/2"
+      >
+        <svg
```

Comment:
What are the `svgs? We have ReactIcons in the project

Reply:
use fa-spinner icon 
![Screenshot (22)](https://user-images.githubusercontent.com/35908401/215264433-0b5823a8-f894-4b63-b578-2e0b931e0730.png)